server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -68,8 +68,11 @@ public InputStream read(BucketName bucketName, BlobId blobId) throws ObjectStore
             .filter(defaultBucket::equals)
             .flatMap(ignored ->
                 Mono.from(cache.read(blobId))
```

Comment:
please do more extraction to avoid all technical steps in this code

Reply:
Yes, i doing it